### ISSUE TITLE
Using Alpine and Webpack in IE11

### ISSUE THREAD

**Issue by keyurshah** [alpinejs/alpine#325]
Hello,

This is more a question on how to use and load Alpine properly.

I'm currently trying to get alpine-ie11.js working alongside an app.js file that is generated using laravel mix (I do this so I can process the code through babel).

This wasn't a problem with the older alpine ie11 version, but the new one with polyfills builtin is having compatibility issues with other libraries that prefer the polyfills to be loaded differently. 

I am loading alpine in the html page and not as an import statement in app.js (because i need alpine-ie11.js)

The problem is that if I load alpine-ie11.js before app.js, it cannot find the function `window.my_alpine_function()`

The way I am outputting any alpine code from app.js is below. I attach the function from inside app.js to the window level so alpine can read it. 

app.js
```
window.alpine_product = alpine_product();
```

html

```
<div x-data=“window.alpine_product”>
</div>
```

basically my dilemna is if I load `alpine-ie11.js` before `app.js` then it cannot find the `window.alpine_product` function.

and if I load `alpine-ie11.js` after `app.js` I get errors from other libraries.

wondering if anyone else has a better way of doing this?

thanks.

**Comment by HugoDF** [comment]
You can try importing alpine-ie11.js from app.js using the path `require('alpinejs/dist/alpine-ie11.js')`

**Comment by keyurshah** [comment]
thanks for your help @HugoDF . I was able to get it to work by rearranging and removing some polyfills i had included for the other libraries.

I also saw this was added recently and this may help as well

```
window.deferLoadingAlpine = callback => {
    // When some condition is met, call callback() and Alpine will load
}
```
